builds/overview-openshift-builds.adoc

```diff
@@ -39,5 +39,5 @@ include::modules/ob-controller-settings.adoc[leveloffset=+2]
 [id="additional-resources_overview-openshift-builds"]
 == Additional resources
 
-* xref:../builds/installing-openshift-builds.adoc#installing-builds-with-operator_installing-openshift-builds[Installing {builds-shortname} by using the console] 
+* xref:../builds/installing-openshift-builds.adoc#installing-builds-with-operator_installing-openshift-builds[Installing {builds-shortname} by using the console]
```

Comment:
Space changed were picked up here?

Reply:
I enabled the settings to strip off white spaces in files. This is why you see this change.